rules/windows/process_creation/proc_creation_win_wmic_recon_unquoted_service_search.yml

```diff
@@ -23,13 +24,8 @@ detection:
         - Image|endswith: '\WMIC.exe'
     selection_cli:
         CommandLine|contains|all:
-            - ' service '
-            - ' get '
-        CommandLine|contains:
-            - 'name'
-            - 'displayname'
-            - 'pathname'
-            - 'startmode'
+            - ' service get '
+            - 'name,displayname,pathname,startmode'
```

Comment:
```suggestion

```
very specific

Reply:
That's the point of the rule. It's meant to be very specific to cover a specific call. The generic version is 68bcd73b-37ef-49cb-95fc-edc809730be6